### DISCUSSION TITLE
osu!lazer should notify the user when score submit is taking a long time

### DISCUSSION THREAD

**Discussion by dumbuz** [ppy/osu#25780] [Ideas]
On bad connections, or when certain errors for occur, score submit at the end of a map will take a long time. osu!lazer gives zero feedback to the user when this is happening. When osu!lazer is attempting to submit a score, the pause button is disabled. So pressing "Back," the pause icon fills up but nothing happens. Obviously, this is confusing.

I believe that when score submit is taking a long time (anything more than a few seconds) a notice should show up saying that osu!lazer is currently submitting a score. Another option would be to suppress this notice until the user attempts to pause/exit the beatmap with the back button.

**Comment by bdach** [comment]
already tracked as https://github.com/ppy/osu/issues/24609 / https://github.com/ppy/osu/issues/23876
